openxla_patches/gpu_build_file.diff

```diff
@@ -0,0 +1,25 @@
+diff --git a/xla/pjrt/gpu/BUILD b/xla/pjrt/gpu/BUILD
+index 9ad1fca31..8ea07ad0a 100644
+--- a/xla/pjrt/gpu/BUILD
++++ b/xla/pjrt/gpu/BUILD
+@@ -237,17 +237,17 @@ cc_library(
+         "@com_google_absl//absl/status",
+         "@tsl//tsl/platform:casts",
+         "@tsl//tsl/platform:errors",
+-    ] + if_cuda([
++    ] + if_cuda_or_rocm([
```

Comment:
this patch looks same like https://github.com/openxla/xla/commit/9938bdbbf34a4098856ef387e3f1c53398d3f1e1, and what is the reason to skip the modify of `load("//xla/stream_executor:build_defs.bzl", "if_cuda_or_rocm", "if_gpu_is_configured")`?

since GPU CI failed with the same issue: `RuntimeError: torch_xla/csrc/device.cpp:72 : Invalid device specification: CUDA:0`, are they related too?

Reply:
No particular reason. 

I started importing on Oct 3 and this change is Oct 4.